runtime(doc): clarify smarttab (#17426)
vim/vim: runtime/doc/insert.txt

cursor.  You will have to delete 'softtabstop' characters first, and then type
extra spaces to get where you want to be.

							*ins-smarttab*
When the 'smarttab' option is on, and the cursor is in Insert mode within an

> indent (i.e. when all the characters before the cursor are either tabs or

Reviewer: ```suggestion
indent (i.e. when there are only blanks in front of the cursor)
```

Author: Ok, changed in the next commit.
